src/test/java/hudson/tasks/junit/SuiteResultTest.java

```diff
@@ -38,6 +38,10 @@
 
 import junit.framework.TestCase;
 
+import static org.hamcrest.core.StringStartsWith.startsWith;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertThat;
+
 /**
```

Comment:
this is causing the [test] (https://jenkins.ci.cloudbees.com/job/plugins/job/junit-plugin/98/org.jenkins-ci.plugins$junit/testReport/hudson.tasks.junit/SuiteResultTest/testSuiteResultPersistence/) to fail , you should modify the test to be aligned with the `org.junit.Assert` API